<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>공격하기</string>
     </attribute>
     <widget class="QPushButton" name="skill1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>81</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="skill2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>81</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="skill3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>71</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>가방</string>
     </attribute>
     <widget class="QPushButton" name="hp_po">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>상처약.png</normaloff>상처약.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="mp_po">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>121</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>PP에이드.png</normaloff>PP에이드.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>도망가기</string>
     </attribute>
     <widget class="QPushButton" name="run">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>251</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>도망가기</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="bettlepokemon">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>180</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>144</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar_1">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(252, 1, 7);
	border-radius: 5px;
 }</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>HP %p</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_3">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(0, 112, 255);
	border-radius: 5px;
 }</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="format">
        <string>HP %p</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="bettlepokemon_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="name_level_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>124</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
	font: 18pt &quot;.AppleSystemUIFont&quot;;
	font: 18pt &quot;.AppleSystemUIFont&quot;;
     border-radius: 5px;
	background-color: rgb(179, 177, 181);
	font: 14pt &quot;American Typewriter&quot;;
 }

 QProgressBar::chunk {
     background-color: rgb(252, 1, 7);
	border-radius: 5px;
 }</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>HP %p</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ment">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>371</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY궁서B</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="name_level">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lavel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
